build-scripts/createRelease.ts

```diff
@@ -41,6 +41,19 @@ for (const changeFile of changeFiles) {
     changelog.entries.push(file)
 }
 
+// tslint:disable-next-line: no-unsafe-any
+changelog.entries.sort((x: {type: string}, y: {type: string}) => {
+    if (x.type === y.type) {
```

Comment:
Would `return x.type.localeCompare(y.type)` work in this case?

Reply:
oh I didn't know that function existed, changed to use localCompare